springmvc-raml-plugin/README.md

```diff
@@ -278,6 +278,11 @@ Then simply include the following code in the POM of the project you wish to gen
 ### baseUri
 (optional) Base URI for generated Spring controllers. This overrules the baseUri attribute from inside the .raml spec.
 
+### controllerMethodThrowsException
+(optional) Parameter which decides if the generated methods in controller have `throws Exception` in the signature.
+This is to use the spring ``@ExceptionHandler` for handling exceptions.
```

Comment:
`@ExceptionHandler` can handle runtime exceptions too. We are adding `throws Exception` in order to support checked exceptions. The reason for that can be existing exception system of checked exceptions in an application that is starting to use the plugin.
So please remove this line.